Formula/openjdk.rb

```diff
@@ -8,8 +8,8 @@ class Openjdk < Formula
     sha256 "29df31b5eefb5a6c016f50b2518ca29e8e61e3cfc676ed403214e1f13a78efd5"
     version "15.0.1"
```

Comment:
```suggestion
    version "15.0.2"
```
This version string also needs to be changed, or else `test-bot` won't be able to find the ARM bottle